docs/content/connectors/postgres-cdc.md

```diff
@@ -0,0 +1,370 @@
+# Postgres CDC Connector
+
+The Postgres CDC connector allows for reading snapshot data and incremental data from PostgreSQL database. This document describes how to setup the Postgres CDC connector to run SQL queries against PostgreSQL databases.
+
+Dependencies
+------------
+
+In order to setup the Postgres CDC connector, the following table provides dependency information for both projects using a build automation tool (such as Maven or SBT) and SQL Client with SQL JAR bundles.
+
+### Maven dependency
+
+```
+<dependency>
+  <groupId>com.ververica</groupId>
+  <artifactId>flink-connector-postgres-cdc</artifactId>
+  <!-- the dependency is available only for stable releases. -->
+  <version>2.2-SNAPSHOT</version>
+</dependency>
+```
+
+### SQL Client JAR
+
+```Download link is available only for stable releases.```
+
+Download [flink-sql-connector-postgres-cdc-2.2-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-postgres-cdc/2.2-SNAPSHOT/flink-sql-connector-postgres-cdc-2.2-SNAPSHOT.jar) and put it under `<FLINK_HOME>/lib/`.
+
+How to create a Postgres CDC table
+----------------
+
+The Postgres CDC table can be defined as following:
+
+```sql
+-- register a PostgreSQL table 'shipments' in Flink SQL
+CREATE TABLE shipments (
+  shipment_id INT,
+  order_id INT,
+  origin STRING,
+  destination STRING,
+  is_arrived BOOLEAN
+) WITH (
+  'connector' = 'postgres-cdc',
+  'hostname' = 'localhost',
+  'port' = '5432',
+  'username' = 'postgres',
+  'password' = 'postgres',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)